lua/nvim-paredit/api/motions.lua

```diff
@@ -64,7 +64,7 @@ local function get_next_node_from_cursor(lang, reversed)
   end
 end
 
-function M.move_to_next_element()
+function M._move_to_next_element(count)
```

Comment:
Can we rename this (and the `prev` version) to something like `move_count_elements_forwards`?